documentation/modules/ROOT/pages/configuration/outbox-event-router.adoc

```diff
@@ -9,26 +13,56 @@
 
 toc::[]
 
+The outbox pattern is a way to ensure that related services that need the same data updates get those updates in a consistent manner. To implement the outbox pattern in a {prodname} application, configure a {prodname} connector to: 
+
+* Capture changes in an outbox table
+* Apply the {prodname} `outbox.EventRouter` single message transformation (SMT)
+
+A {prodname} connector that is configured to apply the outbox SMT should capture changes in only an outbox table. A connector can capture changes in more than one outbox table only if each outbox table has the same structure.  
+
+ifdef::community[]
 [NOTE]
 ====
-This single message transformation (SMT) is under active development right now, so the emitted message structure or other details may still change as development progresses.
+The `outbox.EventRouter` SMT is under active development. The structure of the emitted message or other details might change as development progresses.
 ====
+endif::community[]
 
-[NOTE]
+ifdef::product[]
+[IMPORTANT]
 ====
-This SMT does *not* support the MongoDB connector.
+The {prodname} outbox event router SMT is a Technology Preview feature. Technology Preview features are not supported with Red Hat production service-level agreements (SLAs) and might not be functionally complete; therefore, Red Hat does not recommend implementing any Technology Preview features in production environments. This Technology Preview feature provides early access to upcoming product innovations, enabling you to test functionality and provide feedback during the development process. For more information about support scope, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
 ====
+endif::product[]
+
+See link:https://debezium.io/blog/2019/02/19/reliable-microservices-data-exchange-with-the-outbox-pattern/[Reliable Microservices Data Exchange With the Outbox Pattern] to learn about why the outbox pattern is useful and how it works. 
 
-The Outbox Event Router SMT has the intent to provide out-of-the-box support for the Outbox Pattern.
-For more information on why and how the Outbox Pattern is used please refer our blog post link:/blog/2019/02/19/reliable-microservices-data-exchange-with-the-outbox-pattern/[Reliable Microservices Data Exchange With the Outbox Pattern].
+ifdef::community[]
+For details about how {prodname} implements the outbox pattern, see {jira-url}/browse/DBZ-1169[Outbox SMT Design]. For an example that you can run, see the  link:https://github.com/debezium/debezium-examples/tree/master/outbox[outbox pattern demo], which is in the {prodname} examples repository. It includes an example of how to configure a {prodname} connector to run the `outbox.EventRouter` SMT. 
```

Comment:
Let's remove the reference to the Jira issue, it's not really needed in the docs.

Reply:
Done in the next commit.